Source/USemLogEd/Private/SLEdModeToolkit.cpp

```diff
@@ -229,7 +229,7 @@ FReply FSLEdModeToolkit::GenerateNewSemanticIds()
 		int32 TagIndex = FTags::GetTagTypeIndex(*ActItr, "SemLog");
 		if (TagIndex != INDEX_NONE)
 		{
-			FTags::AddKeyValuePair(ActItr->Tags[TagIndex], "Id", FIds::NewGuidInBase64Url(), true, *ActItr);
+			FTags::AddKeyValuePair(*ActItr, ActItr->Tags[TagIndex].ToString(), "Id", FIds::NewGuidInBase64Url(), true);
```

Comment:
@joserochh this seems a bit unrelated, do you know why were the changed necessary?

Reply:
I had a compilation Error, no function prototype matched this list of arguments so I had to modify  it. Anyway I may keep this local.